src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.csproj

```diff
@@ -24,12 +24,6 @@
     <Compile Include="..\..\..\..\Test\PdbUtilities\Shared\CustomDebugInfoReader.cs">
```

Comment:
Consider adding ` <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file.